virttest/virt_vm.py

```diff
@@ -1472,7 +1472,7 @@ def get_totalmem_sys(self, online='yes', node=''):
                 cmd += "cd /sys/devices/system/node/node%s/;" % node
                 cmd += "for i in `ls -d memory*`;"
             else:
-                cmd = "count=0;cd /sys/devices/system/memory/;for i in `ls`;"
+                cmd = "count=0;cd /sys/devices/system/memory/;for i in `ls -d memory*`;"
```

Comment:
The change will bring impact with previously related case, maybe add some constraints on  version after specific one

Reply:
It will not impact with previous cases except savng the time.